gw_eccentricity/eccDefinition.py

```diff
@@ -1184,6 +1228,180 @@ def interp_extrema(self, extrema_type="pericenters"):
             raise Exception(
                 f"Sufficient number of {extrema_type} are not found."
                 " Can not create an interpolant.")
+        
+    def get_rat_fit(self, x, y):
```

Comment:
hmm, don't like the name...come up with a way to not call the two things get_rat_fit and get_rational_fit? Easily confused.

Reply:
changed `get_rat_fit` to `get_rational_fit_wrapper`